backend/app/usecase/search/search.go

```diff
@@ -31,6 +31,9 @@ func (s Search) Search(query Query, filter Filter) (Result, error) {
 	resultCh := make(chan Result)
 	defer close(resultCh)
 
+	if len(filter.Resources) > len(filter.Orders) {
+		filter.Orders = appendDefaultOrders(filter.Orders, len(filter.Resources)-len(filter.Orders))
```

Comment:
@arberiii , just curious how does this maintain the one to one correspondence of the resource and order? Should we think about extracting resource and order into a separate structure in [Filter](https://github.com/short-d/short/blob/master/backend/app/usecase/search/filter.go).

Reply:
I agree that your suggestion seems better. One mismatch as you mention offline is if the package's client supplies 3 resources and 2 order by's for first and third resource.